lib/lndclient/LndClient.ts

```diff
@@ -103,6 +104,19 @@ class LndClient extends SwapClient {
     return this.chainIdentifier;
   }
 
+  public maximumOutboundCapacity = () => {
+    return this.maximumOutboundAmount;
```

Comment:
Thoughts on an `assert` here to check that the currency passed to the method matches the currency for this client?

Reply:
I'm not sure about this. If we do it here then it would also make sense to do it for the `channelBalance` method.

Reply:
OK good point, fine as is.